
Author: rmh
Description: Use SysV UTMP instead of BSD one (this fixes runtime failure on xdm/kdm/etc)
Status: in upstream BTS

--- xc/programs/xdm/sessreg.c~	2005-02-28 18:35:18.000000000 +0100
+++ xc/programs/xdm/sessreg.c	2005-08-29 23:16:45.000000000 +0200
@@ -74,7 +74,7 @@
 # include	<pwd.h>
 #endif
 
-#if defined(SVR4) || defined(linux)
+#if defined(SVR4) || defined(linux) || defined(__GLIBC__)
 #define SYSV
 #endif
 
